tests/Commands/MakeEnumTest.php

```diff
@@ -0,0 +1,244 @@
+    {
+        $values = [
+            'Monday' => 'monday',
+            'Tuesday' => 'tuesday',
+            'Wednesday' => 'wednesday',
+            'Thursday' => 'thursday',
+            'Friday' => 'friday',
+            'Saturday' => 'saturday',
+            'Sunday' => 'sunday',
+        ];
+
+        $this->runMakeCommand([
+            '--value' => $values,
+            '--formatter' => 'studly',
+        ]);
+
+        $this->assertStringContainsString('const MAP_VALUE = [', $this->content);
+        foreach($values as $method => $value) {
+            $this->assertMethodDocTag($method);
+            $this->assertValueMap($method, $value);
+        }
+    }
+
+    /** @test */
+    public function it_can_make_an_enum_with_values_snake(): void
+    {
+        $values = [
+            'mon_day' => 'mon day',
+            'tues_day' => 'tues day',
+            'wednes_day' => 'wednes day',
+            'thurs_day' => 'thurs day',
+            'fri_day' => 'fri day',
+            'satur_day' => 'satur day',
+            'sun_day' => 'sun day',
+        ];
+
+        $this->runMakeCommand([
+            '--value' => $values,
+            '--formatter' => 'snake',
+        ]);
+
+        $this->assertStringContainsString('const MAP_VALUE', $this->content);
+        foreach($values as $method => $value) {
+            $this->assertMethodDocTag($method);
+            $this->assertValueMap($method, $value);
+        }
+    }
+
+    /** @test */
+    public function it_can_make_an_enum_with_values_const(): void
+    {
+        $values = [
+            'MON_DAY' => 'mon day',
+            'TUES_DAY' => 'tues day',
+            'WEDNES_DAY' => 'wednes day',
+            'THURS_DAY' => 'thurs day',
+            'FRI_DAY' => 'fri day',
+            'SATUR_DAY' => 'satur day',
+            'SUN_DAY' => 'sun day',
+        ];
+
+        $this->runMakeCommand([
+            '--value' => $values,
+            '--formatter' => 'const',
+        ]);
+
+        $this->assertStringContainsString('const MAP_VALUE', $this->content);
+        foreach($values as $method => $value) {
+            $this->assertMethodDocTag($method);
+            $this->assertValueMap($method, $value);
+        }
+    }
+
+    protected function assertMethodDocTag(string $method): void
+    {
+        $this->assertStringContainsString('* @method static self '.$method.'()', $this->content);
+    }
+
+    protected function assertValueMap(string $method, string $value): void
+    {
+        $this->assertStringContainsString('\''.$method.'\' => \''.$value.'\',', $this->content);
+    }
+
+    private function runMakeCommand( array $arguments = []): void
+    {
+        $artisan = $this->artisan('make:enum', array_merge([
+            'name' => 'WeekDay',
+        ], $arguments));
+
+        $artisan->assertExitCode(0);
+        $artisan->expectsOutput(' created successfully.');
+        $artisan->run();
+
+        $this->assertTrue(file_exists(self::WEEKDAY_PATH));
```

Comment:
How about using full files to make these assertions instead of asserting separate lines. Eg.

```php
$this->assertEquals($this->getStub('weekday-a_snake.php'), $output);
```

Reply:
https://github.com/spatie/laravel-enum/pull/12/commits/498d8f97c9cb1909e0f27db0d8c76089553869ae